internal/common/enums.rs

```diff
@@ -345,6 +345,8 @@ macro_rules! for_each_enums {
                 Text,
                 /// The element is a [`ProgressIndicator`](../widgets/progressindicator.md) or behaves like one.
                 ProgressIndicator,
+                /// The role for a [`TextInput`](elements.md#TextInput) element.
```

Comment:
Should that be set on the TextInput or rather on the LineEdit/TextEdit?
It seems it is the later.
So I think this documentation should be fixed:

```suggestion
                /// The role for widget with editable text such as a 
                /// [`LineEdit`](../widgets/lineedit.md) or a [`TextEdit`](../widgets/textedit.md)
```